geometry/cbox.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "geometry/point2d.hpp"
+
+#include <vector>
+
+namespace m2
+{
+// When a size of a convex hull over a set of |points| is less than 4,
+// stores convex hull explicitly, otherwise stores smallest rectangle
+// containing the hull.
+class CBox
+{
+public:
+  CBox(std::vector<PointD> const & points);
+
+  std::vector<PointD> Points() const { return m_points; }
```

Comment:
is it possible to return `std::vector<PointD> const &` here?

Reply:
Sure, done.